client/src/Products/Views.elm

```diff
@@ -242,78 +248,137 @@ cartFormData addToCartForms ( product, variants ) =
 --------------------------------------------------------------
 
 
-detailView : CartForms -> PageData.ProductDetails -> List (Html Msg)
-detailView addToCartForms { product, variants, maybeSeedAttribute, categories } =
+productAttrView selectedItemNumber maybeSeedAttribute =
+    div [ class "tw:flex tw:items-center" ]
+        [ small [ class "text-muted d-block tw:grow" ]
+            [ text <| "Item #" ++ selectedItemNumber ]
+        , htmlOrBlank SeedAttribute.icons maybeSeedAttribute
+        ]
+
+
+variantSelectView paddingClass variantSelect isOutOfStock =
```

Comment:
Does the `outOfStock` badge currently consider the whole product or just a chosen variant?

The old behaviour used to show this badge only when _all_ variants were out of stock.
See [here](https://github.com/Southern-Exposure-Seed-Exchange/southernexposure.com/blob/7a0b875e8f6db2672f7fbf03e1950fa86b824715/client/src/Product.elm#L134)

Currently, the idea is to show badge for each variant separately.